rampwf/utils/submission.py

```diff
@@ -97,7 +97,8 @@ def train_test_submission(problem, module_path, X_train, y_train, X_test=None,
     except Exception:
         print_submission_exception(save_output, output_path)
         set_state('training_error', save_output, output_path)
-        exit(1)
+        # exit(1)
```

Comment:
Ups. This is something that I changed when debugging the code with you. Could you revert this back